app/src/components/main/breadcrumbs/breadcrumb/breadcrumb.scss

```diff
@@ -13,3 +13,15 @@
     height: 20px;
   }
 }
+
+.breadcrumb:nth-child(even) {
+  display: block;
+  word-wrap: break-word;
+  width: 27.5%;
+}
+
+@media screen and (max-width: 425px) {
```

Comment:
Too many magical numbers in this PR. Is it possible to use only variables from `common/css/variables/screen-size-variable.scss`?

Reply:
fixed